### PR TITLE
fix(ci): kill and wait for tmux sessions to shut down

### PR DESCRIPTION
**Pull request by fuyufjh**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

See https://github.com/risingwavelabs/risingwave/pull/11520/files#r1286632767

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR contains user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Review by huangjw806 — APPROVED** [review]

**Comment by huangjw806** [comment]
Still doesn't look good.
image

**Comment by kwannoel** [comment]
Fixing it, looks like:
```
        // TODO: join dashboard service back to local thread.
        tokio::spawn(dashboard_service.serve(address_info.ui_path));
```

in `src/meta/src/rpc/server.rs`.

**Comment by fuyufjh** [comment]
> Fixing it, looks like:
> 
> ```
>         // TODO: join dashboard service back to local thread.
>         tokio::spawn(dashboard_service.serve(address_info.ui_path));
> ```
> 
> in `src/meta/src/rpc/server.rs`.
> 
> I guess the child process did not exit.

I don't think it's the cause. It's a tokio task instead of child process 🥹

File: Makefile.toml [hunk 1]
```diff
@@ -1077,6 +1077,9 @@ while true; do
     fi
 done
 echo "Tmux windows are now empty."
+
+# FIXME: a dirty workaround for #11519.
```
[the remark this answers is not in the document]
Comment (huangjw806) [reply Makefile.toml:1081]: Is 5 seconds too long? The difference in machine performance is not that big.

[the remark this answers is not in the document]
Comment (fuyufjh) [reply Makefile.toml:1081]: Just a provisional fix. I just want to make the CI works ASAP 😭

I guess the root cause is that the tmux sessions were over but the processes didn't get killed, but I don't know how to check whether all RW processes has quited. We should fix it in this week.

[the remark this answers is not in the document]
Comment (huangjw806) [reply Makefile.toml:1081]: It seems that sleep 5 cannot ensure that all processes exit.🥵

**Comment by fuyufjh** [comment]
Okay, now it works.


But the meta-node did take quite a long time (around 1 min) to exit...

https://buildkite.com/risingwavelabs/pull-request/builds/29205#0189d4a9-c5c1-4595-94c0-560bfc1d5593/2115-2130


```
2023-08-08 10:27:59 UTC	[cargo-make] INFO - Running Task: wait-processes-exit
2023-08-08 10:27:59 UTC	Process meta-node is still running... Wait for 1 sec
2023-08-08 10:28:00 UTC	Process meta-node is still running... Wait for 1 sec
.....
2023-08-08 10:28:55 UTC	Process meta-node is still running... Wait for 1 sec
2023-08-08 10:28:56 UTC	Process meta-node is still running... Wait for 1 sec
2023-08-08 10:28:57 UTC	All processes has exited.
```

**Pull request merged**